modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -66,6 +66,88 @@ If `storageType` is not set and the `transportHost` uses HTTP, the PTP daemons a
 $ oc apply -f ptp-operatorconfig.yaml
 ----
 
```

Comment:
If dynamic volume provisioning is supported in the cluster, a PersistentVolume will be automatically created upon creation of PVC, otherwise a PersistentVolume needs to be manually created to meet the demond of PVC. For example:
...
(see: https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/)

Reply:
Updated. PTAL.